flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -306,8 +307,8 @@ int32_t transmitter_control_update()
 	    validChannel[MANUALCONTROLSETTINGS_CHANNELGROUPS_FLIGHTMODE];
 
 	// because arming is optional we must determine if it is needed before checking it is valid
-	bool arming_valid_input = settings.ChannelGroups[MANUALCONTROLSETTINGS_CHANNELGROUPS_ARMING] >= MANUALCONTROLSETTINGS_CHANNELGROUPS_NONE ||
-	    validChannel[MANUALCONTROLSETTINGS_CHANNELGROUPS_ARMING];
+	bool arming_valid_input = (settings.Arming >= MANUALCONTROLSETTINGS_ARMING_SWITCH) ||
```

Comment:
could we do this in a way that doesn't depend on the numerical value on the enum? that makes me nervous in case we ever reorder things.